lib/Logger/LoggerFeature.cpp

```diff
@@ -128,6 +128,10 @@ void LoggerFeature::collectOptions(std::shared_ptr<ProgramOptions> options) {
   options->addHiddenOption("--log.force-direct",
                            "do not start a seperate thread for logging",
                            new BooleanParameter(&_forceDirect));
+
+  options->addHiddenOption("--log.full-url",
+                           "log the full url of a HTTP, if false only log upto '?'",
```

Comment:
"log the full URL for HTTP requests, if false only log up to '?`"

Reply:
what about: `--log.request-parameters` ?
Also i think this could be confusing.
Description is like:

`log the full Request, if set to false all parameters (query, body and url) will not be logged`